mcodingbot/plugins/python.py

```diff
@@ -2,6 +2,7 @@
 
 import crescent
 import hikari
+import hikari.impl
```

Comment:
typehints should use the public models, which I believe is `hikari.ActionRowBuilder`

Reply:
that doesn't exist